fusion-plugin-universal-logger/src/tokens.js

```diff
@@ -6,11 +6,11 @@
  * @flow
  */
 
-import type {$winstonLoggerConfig} from 'winston';
+import type {LoggerConfig} from 'winston';
```

Comment:
These types weren't previously exported by the `winston` libdef, and we had made custom edits to export the raw private types.

Consumers that use our altered libdefs will want to make this same change.